src/pages/guides/using/creating_rest_apis.md

```diff
@@ -153,6 +153,145 @@ curl -i -X OPTIONS https://adobeioruntime.net/...
 
 ## Securing the API endpoints
 
+### Oauth (using the Adobe Identity Management System)
+
+You can configure an action to require IMS validation for incoming requests using the following command: 
+```bash
+wsk action update <action_name> --web true -a require-validation true
+```
+
+Once IMS authentication has been enabled for an action, the only way to allow access to the action is by specifying a list of IMS scopes or client IDs that are permitted to invoke the action. 
+
+The following code snippet demonstrates how to configure access using a standard Swagger file and the `security` object:
+
+```json
+{
+    "basePath": "/v2",
+    "paths": {
+      "/ims-secure-endpoint": {
+        "get": {
+          "operationId": "your-namespaces/default/my-ims-secure-web-action.json",        
+          "security": [
+            {
+              "scopes_auth": {
+                "scopes" : [
+                  "write:pets",
+                  "read:pets"
+                ]
+              }
+            }
+          ]
+        }
+      }
+    },
+    "securityDefinitions": {
+        "scopes_auth": {
+          "type": "oauth2",
+          "authorizationUrl": "https://ims-na1-stg1.adobelogin.com/ims/validate_token/v1?client_id=ioruntime&type=access_token",
+          "flow": "implicit",
+          "scopes": {
+            "write:pets": "modify pets in your account",
+            "read:pets": "read your pets"
+          }
+        }
+    }
+}
+```
+This enables scope validation for the API endpoint, allowing requests with access tokens that have the scopes `write:pets` OR `read:pets`.
+>Ensure that there is a `securityDefinitions` defined with type `oauth2`. You can use any authorization provider by providing a `authorizationUrl`, however if left empty, IMS, will be used as default.
+
+After publishing the Swagger file, you can use this endpoint to call the action `your-namespaces/default/my-require-validation-web-action` as follows: 
+```bash
+curl -i -H "Authorization: Bearer <ims_access_token>" https://guest.adobeioruntime.net/api/v2/ims-validation-endpoint
```

Comment:
How does the client obtain the `ims_access_token`? I assume when using aio CLI, it's available there?
Maybe there is another document that we can link here that describes this.

Reply:
This is standard oauth2 validation. The customer needs to register with an oauth provider prior to configuring any of this. IMS represents Adobe's solution, but it could be any other: https://en.wikipedia.org/wiki/List_of_OAuth_providers
I'm not sure we should get into how to register a client with IMS as this could get convoluted.

Reply:
Maybe we can add a link to another resource that describes this? I'm not sure everything will be obvious for all developers that work with Runtime.